_pages/author-priyanshur.html

```diff
@@ -0,0 +1,28 @@
+---
+title: "Priyanshu Raj"
+layout: default
+permalink: "/author-priyanshu-raj"
```

Comment:
The value of permalink should match with the file name

## Changes to make
1. Either make the permalink from `permalink: "/author-priyanshu-raj"` to `permalink: "/author-priyanshur`
2. Or change the filename from ` _pages/author-priyanshur.html` to ` _pages/author-priyanshu-raj.html`

Reply:
@AvikantSrivastava changes done, please once review

Reply:
@kaushikikrityy okay checking!

Reply:
@AvikantSrivastava @kaushikikrityy changes done, please check once

Reply:
@AvikantSrivastava  yeah sorry! , I think i have done it partially, gemfile.lock has been deleted as told by you , the other one is not done. Could you please assure me once.